core/domain/email_manager.py

```diff
@@ -299,11 +367,39 @@ def send_post_signup_email(user_id):
 
 
 def require_valid_intent(intent):
+    """Checks if the given intent is valid, raises an exception if it is not.
+
+    Args:
+        intent: str. The intent string (cause/purpose) of the email.
+
+    Returns:
+        None
+
+    Raises:
+        Exception: The given intent did not match an entry in
+                    feconf.VALID_MODERATOR_ACTIONS
+    """
+
     if intent not in feconf.VALID_MODERATOR_ACTIONS:
         raise Exception('Unrecognized email intent: %s' % intent)
 
 
 def _get_email_config(intent):
+    """Return the default body for the email type
```

Comment:
Could you move some words from the next line to the end of this one until you hit the 80-char limit? This line seems oddly short.